pkg/kubelet/kubelet.go

```diff
@@ -60,6 +60,7 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/util/bandwidth"
 	utilErrors "k8s.io/kubernetes/pkg/util/errors"
+	kubeioutil "k8s.io/kubernetes/pkg/util/io"
```

Comment:
We preface pretty much everything we import from Kube in OpenShift with a
k.  It gets very k'y.  `kapi`, `kclientcmd`, `kutil`. I'm not advocating
for reverse order other than ioutil is pretty well known.

On Mon, Sep 14, 2015 at 11:19 PM, Tim Hockin <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/kubernetes/kubernetes/pull/13791#discussion_r39471034>
> :
>
> > @@ -60,6 +60,7 @@ import (
> >  	"k8s.io/kubernetes/pkg/util"
> >  	"k8s.io/kubernetes/pkg/util/bandwidth"
> >  	utilErrors "k8s.io/kubernetes/pkg/util/errors"
> > +	kubeioutil "k8s.io/kubernetes/pkg/util/io"
>
> our pattern for pkg/util/foo, where "foo" overlaps with stdlib, is usually
> "utilfoo". Apparently underscores are verboten (so much for a C-like
> language) because "util_foo" seems much more approachable to me. We could
> instead say kutilfoo or kubefoo or something.
> … <#14fcf040c12969a2_>
> On Mon, Sep 14, 2015 at 7:20 PM, Clayton Coleman <notifications@github.com>
> wrote: In pkg/kubelet/kubelet.go <
> https://github.com/kubernetes/kubernetes/pull/13791#discussion_r39468631>
> : > @@ -60,6 +60,7 @@ import ( > "k8s.io/kubernetes/pkg/util" > "
> k8s.io/kubernetes/pkg/util/bandwidth" > utilErrors "
> k8s.io/kubernetes/pkg/util/errors" > + kubeioutil "
> k8s.io/kubernetes/pkg/util/io" ioutil to match the stdlib? minor point
> really. … <#14fcecdd9afe7449_> On Mon, Sep 14, 2015 at 9:17 PM, Tim Hockin <
> notifications@github.com> wrote: In pkg/kubelet/kubelet.go <
> https://github.com/kubernetes/kubernetes/pull/13791#discussion_r39465928>
> : > @@ -60,6 +60,7 @@ import ( > "k8s.io/kubernetes/pkg/util" > "
> k8s.io/kubernetes/pkg/util/bandwidth" > utilErrors "
> k8s.io/kubernetes/pkg/util/errors" > + kubeioutil "
> k8s.io/kubernetes/pkg/util/io" utilio ? — Reply to this email directly or
> view it on GitHub <
> https://github.com/kubernetes/kubernetes/pull/13791/files#r39465928>. --
> Clayton Coleman | Lead Engineer, OpenShift — Reply to this email directly
> or view it on GitHub <
> https://github.com/kubernetes/kubernetes/pull/13791/files#r39468631>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13791/files#r39471034>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift